orgchart/import_orgchart_data.sh

```diff
@@ -1,9 +1,4 @@
 #!/bin/bash
-go run import_csv.go "extracted/2010-2020/gazette-2006-1-2.csv"
-go run import_csv.go "extracted/2010-2020/gazette-2006-1-3.csv"
-go run import_csv.go "extracted/2010-2020/gazette-2006-1-4.csv"
-go run import_csv.go "extracted/2010-2020/gazette-2006-1-5.csv"
-
 
 go run import_csv.go "extracted/2010-2020/gazette-2010-4-30.csv"
```

Comment:
We may not need to manually add the filenames to this script. You can write a bash script that will first get a list of files and then in a for loop run the command on each of them